decidim-core/lib/decidim/db/common-passwords.txt

```diff
@@ -3189,7 +3189,7 @@ dragonking
 dragon6661
 dragon2000
 doroshenko
-don'tremember
+do notremember
```

Comment:
Are we sure, we want to modify this file ?

Reply:
No, we don't want to change that. Fixed by 1a7b342.